index.html

```diff
@@ -5,11 +5,18 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
- [ ] Please make sure that your design is responsive  
- the about section is not positioned correctly when the user moves to the desktop version the section just goes down 
- there is a distance that appear between the projects when the user use the desktop or wider and wider screens I think you should use some flexible sizes for the project sizes such as `vw` and use grid to align the item (this is just a suggestion) the grid is the best way to make these card same as the design since grid are better for two dimensional alignment
![image](https://github.com/norbiboy1/Portfolio/assets/74744787/b5fe3235-0564-4e53-8d90-ea723b1c2e84)
![image](https://github.com/norbiboy1/Portfolio/assets/74744787/c020729d-f1cb-4dfe-9bc5-4416f2d65637)